extensions-contrib/opentelemetry-emitter/src/test/java/org/apache/druid/emitter/opentelemetry/OpenTelemetryEmitterTest.java

```diff
@@ -0,0 +1,165 @@
+                                            DateTimes.nowUtc(),
+                                            "query/cache/total/hitRate",
+                                            0.54
+                                        )
+                                        .build(
+                                            "broker",
+                                            "brokerHost1"
+                                        );
+
+    OpenTelemetryEmitter emitter = EasyMock.createMock(OpenTelemetryEmitter.class);
+    emitter.emit(notQueryTimeMetric);
+    EasyMock.replay(emitter);
+
+    emitter.emit(notQueryTimeMetric);
+    EasyMock.verifyUnexpectedCalls(emitter);
+  }
+
+  @Test
+  public void testTraceparentId()
+  {
+    final String traceId = "00-54ef39243e3feb12072e0f8a74c1d55a-ad6d5b581d7c29c1-01";
+    final String expectedParentTraceId = "54ef39243e3feb12072e0f8a74c1d55a";
+    final String expectedParentSpanId = "ad6d5b581d7c29c1";
+    final Map<String, String> context = new HashMap<>();
+    context.put("traceparent", traceId);
+
+    final String serviceName = "druid/broker";
+    final DateTime createdTime = DateTimes.nowUtc();
+    final long metricValue = 100;
+
+    NoopExporter noopExporter = new NoopExporter();
+    SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
+                                                        .addSpanProcessor(SimpleSpanProcessor.create(noopExporter))
+                                                        .build();
+
+    final ServiceMetricEvent queryTimeMetric =
+        new ServiceMetricEvent.Builder().setDimension("context", context)
+                                        .build(
+                                            createdTime,
+                                            "query/time",
+                                            metricValue
+                                        )
+                                        .build(
+                                            serviceName,
+                                            "host"
+                                        );
+    OpenTelemetrySdk.builder()
+                    .setTracerProvider(tracerProvider)
+                    .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
+                    .buildAndRegisterGlobal();
+
+    OpenTelemetryEmitter emitter = new OpenTelemetryEmitter();
+    emitter.emit(queryTimeMetric);
+
+    Assert.assertEquals(1, noopExporter.spanData.size());
+
+    SpanData actualSpanData = noopExporter.spanData.iterator().next();
+    Assert.assertEquals(serviceName, actualSpanData.getName());
+    Assert.assertEquals((createdTime.getMillis() - metricValue) * 1_000_000, actualSpanData.getStartEpochNanos());
+    Assert.assertEquals(expectedParentTraceId, actualSpanData.getParentSpanContext().getTraceId());
+    Assert.assertEquals(expectedParentSpanId, actualSpanData.getParentSpanContext().getSpanId());
```

Comment:
It's usually a best practice to add a static import for `import static org.junit.Assert` 
```suggestion
    assertEquals(serviceName, actualSpanData.getName());
    assertEquals((createdTime.getMillis() - metricValue) * 1_000_000, actualSpanData.getStartEpochNanos());
    assertEquals(expectedParentTraceId, actualSpanData.getParentSpanContext().getTraceId());
    assertEquals(expectedParentSpanId, actualSpanData.getParentSpanContext().getSpanId());
```

Reply:
The Druid codestyle doesn't allow static imports